Content.Shared/Chemistry/Components/Solution.cs

```diff
@@ -27,6 +27,9 @@ public sealed partial class Solution : IEnumerable<ReagentQuantity>, ISerializat
         [ViewVariables]
         public FixedPoint2 Volume { get; set; }
 
+        [ViewVariables(VVAccess.ReadWrite)]
+        public bool IsSeparatedByLayers { get; set; }
```

Comment:
This has to be a datafield

Reply:
fixed

Reply:
This needs documenting so it's clear what it even does.